xbmc/video/VideoDatabase.cpp

```diff
@@ -443,10 +454,23 @@ void CVideoDatabase::CreateViews()
                                      "    episode.idShow = seasons.idShow AND episode.c%02d = seasons.season"
                                      "  JOIN files ON"
                                      "    files.idFile = episode.idFile "
-                                     "GROUP BY seasons.idSeason",
+                                     "GROUP BY seasons.idSeason,"
+				     "         seasons.idShow,"
+				     "         seasons.season,"
+				     "         seasons.name,"
+				     "         seasons.userrating,"
+				     "         tvshow_view.strPath,"
+				     "         tvshow_view.c%02d,"
+				     "         tvshow_view.c%02d,"
+				     "         tvshow_view.c%02d,"
+				     "         tvshow_view.c%02d,"
+				     "         tvshow_view.c%02d,"
+				     "         tvshow_view.c%02d ",
                                      VIDEODB_ID_TV_TITLE, VIDEODB_ID_TV_PLOT, VIDEODB_ID_TV_PREMIERED,
                                      VIDEODB_ID_TV_GENRE, VIDEODB_ID_TV_STUDIOS, VIDEODB_ID_TV_MPAA,
-                                     VIDEODB_ID_EPISODE_AIRED, VIDEODB_ID_EPISODE_SEASON);
+                                     VIDEODB_ID_EPISODE_AIRED, VIDEODB_ID_EPISODE_SEASON,
+                                     VIDEODB_ID_TV_TITLE, VIDEODB_ID_TV_PLOT, VIDEODB_ID_TV_PREMIERED,
+                                     VIDEODB_ID_TV_GENRE, VIDEODB_ID_TV_STUDIOS, VIDEODB_ID_TV_MPAA);
```

Comment:
dupliciate of line 470

Reply:
Yes, the below variables are used again 

![image](https://user-images.githubusercontent.com/23155784/50663820-9bdc9d00-0fb3-11e9-9fff-d00a9e3ec06c.png)

in order to cover the additional references in the group by clause:

![image](https://user-images.githubusercontent.com/23155784/50663860-bb73c580-0fb3-11e9-98dd-9a94356b30e3.png)

As all individual columns that are not part of aggregate function in the SELECT clause must also be in GROUP BY clause:

![image](https://user-images.githubusercontent.com/23155784/50663895-e100cf00-0fb3-11e9-9978-a77cfcdadf5a.png)

I hope it makes sense. I assume that the pre-processor will substitute each reference of c%02d starting from top to bottom with one variable starting from left to right and top to bottom...

Please correct my understanding if I'm wrong...

Thanks and regards

Reply:
Sorry, my SQL knowledge is very limited.
I just noticed them and thought it was an issue caused by squashing commits.

I believe you if you say it's required.

Reply:
No worries, many thanks for your checks. 
Sanity test ok. New views are now able to be queried:

![image](https://user-images.githubusercontent.com/23155784/50677265-98700280-1001-11e9-90d6-77cfe9627598.png)

Moving on to small scale UAT now, re-creating my video library from Trakt and running Trakt addon to see if those views still deliver the functionality Trakt expects or whether we have broken anything in terms of functionality...

This will take about an hour or so...(my library is big...)

Reply:
Views seem to be working...

![image](https://user-images.githubusercontent.com/23155784/50679389-da05ab00-100b-11e9-8505-0b1065a1ea8d.png)

![image](https://user-images.githubusercontent.com/23155784/50679413-e8ec5d80-100b-11e9-8901-40eb4beb3b35.png)

Reply:
so scrobbling and adding data to library seems to be working fine, nothing seems to be broken so far...waiting to see trakt synchronization afterwards...

Reply:
> ot: PrepareSQL works internally like printf you could be able use numbered argument to remove duplicity

I don't think that is possible. Plain printf and the custom implementation for `PrepareSQL` does not support numbering arguments.